hedera-node/configuration/dev/application.properties

```diff
@@ -2,6 +2,7 @@ balances.exportDir.path=data/accountBalances/
 upgrade.artifacts.path=data/upgrade
 contracts.chainId=298
 contracts.maxGasPerSec=15000000000
+contracts.systemContract.tokenInfo.v2.enabled=true
```

Comment:
we should probably change these to be false for now.

Reply:
Agreed, but we could keep dev config so that we won't have to rework the tests?

Reply:
yes, that would be fine.